HowToMLFlow.md

```diff
@@ -0,0 +1,234 @@
+# HowTo MLFlow
+
+This is a tutorial on how to make a quick pipeline and get it up and running.
+
+What we are gonna do in this tutorial is to download a dataset and then log it as an artifact.
+
+Afterwards we are gonna read it from another step and then print the first 10 lines.
+
+## Table of contents
+
+- [HowTo MLFlow](#howto-mlflow)
+  - [Table of contents](#table-of-contents)
+  - [SETUP](#setup)
+  - [CREATING THE PIPELINE](#creating-the-pipeline)
+    - [What does a pipeline consist of?](#what-does-a-pipeline-consist-of)
+    - [Creating the steps](#creating-the-steps)
+    - [Content for data_ingestion step](#content-for-data_ingestion-step)
+    - [Making a conda_env](#making-a-conda_env)
+    - [MLproject FILE](#mlproject-file)
+    - [Content for data_display step](#content-for-data_display-step)
+  - [RUNNING THE PIPELINE](#running-the-pipeline)
+    - [RUNNING THE UI](#running-the-ui)
+
+## SETUP
+
+To get started you will need 2 things.
+
+Nr. 1**Anaconda installed** (used to install python and libraries), you can find the guide on how to install: [here](https://www.datacamp.com/tutorial/installing-anaconda-windows)
+
+Nr. 2 **MLFlow installed.** The easiest way to install MLFlow is searching for: Anaconda Prompt (Anaconda3) , then clicking enter and making sure the Anaconda Prompt is open, after that you can simply use this pip command to install MLFlow into your Anaconda Environment:
+
+`pip install mlflow`
+
+## CREATING THE PIPELINE
+
+### What does a pipeline consist of?
+
+A pipeline consists of the **steps**, a **conda environment** file/or already made environment and in most cases an **MLproject file**.
```

Comment:
MLproject file isn't always necessary, mlflow will work fine without an MLproject file. You will just lose control over how to specifically define steps for MLFlow and it might interpret them wrong by itself